diff --git a/conf/layer.conf b/conf/layer.conf
index 606f0dd..ffee86d 100644
--- a/conf/layer.conf
+++ b/conf/layer.conf
@@ -9,7 +9,7 @@ BBFILE_PATTERN_rust-layer := "^${LAYERDIR}/"
 BBFILE_PRIORITY_rust-layer = "7"
 
 LAYERDEPENDS_rust-layer = "core openembedded-layer"
-LAYERSERIES_COMPAT_rust-layer = "gatesgarth hardknott honister kirkstone mickledore scarthgap"
+LAYERSERIES_COMPAT_rust-layer = "gatesgarth hardknott honister kirkstone mickledore scarthgap styhead"
 
 # Override security flags
 require conf/distro/include/rust_security_flags.inc
